python/activator/middleware_interface.py

```diff
@@ -144,6 +165,13 @@ def prep_butler(self, visit: Visit) -> None:
         shutil.rmtree(visit_dir, ignore_errors=True)
 
     def ingest_image(self, oid: str) -> None:
+        """Ingest an image into the temporary butler.
+
+        Parameters
+        ----------
+        oid : `str`
+            Identifier for incoming image.
```

Comment:
Might be worth documenting the format for the `oid` -- this seems to be the full bucket path to the image.